Init update (#1)
sweetmantech/docs-in-process: docs/pages/getting-started.mdx

# Getting Started

> Welcome to InProcess! This guide will help you get started with our protocol.

Reviewer: ```suggestion
Welcome to In Process! This guide will help you get started with our protocol.
```

## Quick Start

1. Visit our website at [https://www.inprocess.fun/](https://www.inprocess.fun/)

> 2. Connect your wallet

Reviewer: We want to avoid technical / onchain terms to be more accessible to mainstream creators.
```suggestion
2. Sign In
```

> 3. Mint your first token through our user-friendly interface

Reviewer: ```suggestion
3. Create your first moment through our web interface
```
